ansible/playbooks/roles/grafana/tasks/upgrade/password_change.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Fail when grafana admin password isn't set
+  fail:
+    msg: "Please specify grafana admin password (grafana_security.admin_password)"
```

Comment:
Even if it is default password it's better not to print this. Maybe we can write there Please specify grafana admin password different from the default one.